core/src/main/java/com/netease/arctic/catalog/IcebergCatalogWrapper.java

```diff
@@ -330,6 +332,11 @@ public BasicIcebergTable(
       super(tableIdentifier, icebergTable, arcticFileIO, catalogProperties);
     }
 
+    @Override
+    public String name() {
+      return icebergTable.name();
```

Comment:
The superclass has implemented the `name` method, ref: https://github.com/NetEase/amoro/blob/master/core/src/main/java/com/netease/arctic/table/BasicUnkeyedTable.java#L119

The logging error is caused by some classes in Iceberg that directly use `Table#toString` to obtain the name of the table when logging, so it is necessary to implement the `toString` method in `BasicUnkeyedTable`, ref:https://github.com/apache/iceberg/blob/main/core/src/main/java/org/apache/iceberg/SnapshotScan.java#L126

Reply:
OK. It seems that it's already fixed by #1914. But how about the absent serialization ID?

Reply:
It seems that there won't be multiple versions of the class in the runtime environment, so it's acceptable to not set a serialization ID.